apps/ledger-live-desktop/tests/specs/speculos/subAccount/receive.subAccount.spec.ts

```diff
@@ -0,0 +1,47 @@
+import test from "../../../fixtures/common";
+import { Token } from "../../../enum/Tokens";
+import { specs } from "../../../utils/speculos";
+import { Application } from "tests/page";
+import { addTmsLink } from "tests/fixtures/common";
+
+const tokens: Token[] = [
+  Token.ETH_USDT,
+  Token.ETH_LIDO,
+  Token.TRON_USDT,
+  Token.TRON_BTT,
+  Token.BSC_BUSD,
+  Token.BSC_SHIBA,
+  Token.MATIC_DAI,
+  Token.MATIC_UNI,
+];
+
+//Reactivate test after fixing the GetAppAndVersion issue - Jira: LIVE-12581
+for (const [i, token] of tokens.entries()) {
+  test.describe.skip("Receive @smoke", () => {
```

Comment:
why tests are disabled? didn't we say that with retry they should pass? they are passing on my side

Reply:
They are also passing on my side (localy) but not in the CI : https://github.com/LedgerHQ/ledger-live/actions/runs/9599431113/job/26473139616